packages/office-ui-fabric-react/src/components/Button/SplitButton/SplitButton.styles.ts

```diff
@@ -143,6 +146,12 @@ export const getStyles = memoizeFunction(
                 backgroundColor: 'Window'
               }
             }
+          },
+          [HighContrastSelector]: {
+            border: `1px solid ${palette.neutralSecondaryAlt}`,
```

Comment:
Actually, why are you changing the border color here and again two lines below? You could specify `borderStyle` and `borderWidth` separately to not do this.